python/smqtk/web/search_app/modules/iqr/iqr_search.py

```diff
@@ -300,13 +281,16 @@ def iqr_session_info():
                         tuple(d.uuid() for d in iqrs.negative_descriptors),
 
                     # UUIDs of example positive descriptors
-                    "ex_pos": tuple(self._iqr_example_pos_descr[iqrs.uuid]),
-                    "ex_neg": (),  # No user negative examples supported yet
+                    "external_pos": tuple(self._iqr_example_pos_descr[iqrs.uuid]),
+                    "external_neg": tuple(self._iqr_example_pos_descr[iqrs.uuid]),
```

Comment:
This should probably map to an empty tuple instead of positive UUIDs. Negative external queries are not supported, so I support it does not matter either way...